### PR TITLE
Removed Pulkit Bajaj

### LINKED ISSUE #1163: Remove Contributor
Remove Pulkit Bajaj from Marketing team page

### PR DESCRIPTION
**Pull request by singh-sarabjeet**
Fixes #1163

### CODE REVIEW

File: src/data/teams.json [hunk 1]
```diff
@@ -738,18 +738,18 @@
...
-      }
+          "title": "Blog Project Manager",
+          "isLead": true,
+          "payPerDay": 2800,
+          "createdDate": "2020-07-08T02:14:59.307535Z",
+          "contributor": {
+            "contributorId": "63",
+            "displayName": "Ijeoma Okoh",
+            "githubUsername": "Aiijay",
+            "profileImage": "https://ca.slack-edge.com/T07PJD1FZ-U01EBD81EHG-43d3e010ec4a-512",
```
Comment (jamessspanggg) [inline src/data/teams.json:749]: can u take his github image url instead of his slack image url?

Comment (singh-sarabjeet) [reply src/data/teams.json:749]: sure

**Comment by singh-sarabjeet** [comment]
TNB account: c5778aee3cfd83c2088ae81c178c3d34931399f20c1492c33044d8cd85e21898

**Comment by jamessspanggg** [comment]
> It is strictly advised to create a single PR per issue. In any case, where two or more issues have the same root cause, you can club them in a single PR, otherwise, refrain from doing so.

@singh-sarabjeet can you separate the PRs?

**Comment by singh-sarabjeet** [comment]
Sure I'll split them

**Review by jamessspanggg — APPROVED** [review]

**Review by thesanjeevsharma — APPROVED** [review]

**Pull request merged**